libs/basics/src/assert.ts

```diff
@@ -15,7 +15,7 @@ export function assert(
  * Asserts that `condition` is true. This function exists to avoid a polyfill
  * for `assert` in the browser.
  */
-export function assertDefined<T>(entity?: T, message?: string): T {
+export function assertDefined<T>(entity?: T | null, message?: string): T {
```

Comment:
This change is relevant for subsequent commits, as the Google Cloud API types include a number of values that can sometimes be `null`. And before this change, calling `assertDefined` on these values wasn't properly communicating to TS that they weren't `null`.